enclave/ecalls.cpp

```diff
@@ -85,16 +85,16 @@ int enclave_XGBoosterEvalOneIter(BoosterHandle handle, int iter, DMatrixHandle d
   return XGBoosterEvalOneIter(handle, iter, dmats, (const char**) eval_names, len, (const char**) out_str);
 }
 
-int enclave_XGBoosterLoadModel(BoosterHandle handle, const char *fname) {
+int enclave_XGBoosterLoadModel(BoosterHandle handle, const char *fname, char * username) {
```

Comment:
We'd like to keep consistent formatting throughout our C++ code -- please change the argument in function signature here and in all other relevant places to either `char* username` or `char *username`

Reply:
done good point